packages/bot/src/components/flyout.jsx

```diff
@@ -0,0 +1,33 @@
+import React            from 'react';
+import PropTypes        from 'prop-types';
+import { connect }      from '../stores/connect';
+import                       '../assets/sass/scratch/_flyout.scss';
```

Comment:
@aaron-binary i think we are going to rename `_` in our naming. we follow trader